lib/smile-identity-core/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 module SmileIdentityCore
-  VERSION = '2.2.5'
+  VERSION = '3.0.0'
```

Comment:
Since this has no breaking changes, we can probably move to 2.3.0 instead. What do you think?

Reply:
Done